homeassistant/components/xiaomi_miio/light.py

```diff
@@ -800,9 +772,9 @@ async def async_update(self):
 class XiaomiPhilipsMoonlightLamp(XiaomiPhilipsBulb):
     """Representation of a Xiaomi Philips Zhirui Bedside Lamp."""
 
-    def __init__(self, name, light, model, unique_id):
+    def __init__(self, name, device, entry, unique_id):
         """Initialize the light device."""
-        super().__init__(name, light, model, unique_id)
+        super().__init__(name, device, entry, unique_id)
 
         self._hs_color = None
         self._state_attrs.pop(ATTR_DELAYED_TURN_OFF)
```

Comment:
There's an attribute called `ATTR_BRAND` here. What's that? Sounds like static info.

Reply:
It is some sort of boolean (true/false), I actually have no idea what it does or what it is, I don't own such a device, just converting everything to config flow.

I found some info about it here:
https://github.com/rytilahti/python-miio/blob/dfda13edb8bcfd844b2e1f2d7d7374a9a0709ac6/miio/philips_moonlight.py#L74-L82
https://python-miio.readthedocs.io/en/stable/api/miio.philips_moonlight.html#miio.philips_moonlight.PhilipsMoonlight

But not very clear for me.
I did not make the ATTR_BRAND and it does not seem static so I don't really want to remove it if I don't know what it is.
But if you say so I am happy to just remove it.

Reply:
This attribute isn't static. It indicates a Mi Band is paired with the devices (true/false) and something like "the customer of the mi band is sleeping or not". ;-)

Reply:
Is it a misspelling then `brand` -> `band`?

Reply:
I think so, I will adjust